.github/workflows/ci_linux.yml

```
@@ -44,10 +52,29 @@ jobs:
           inputs.use_prebuilt_artifacts == 'true'
         )
       }}
-    strategy:
-      fail-fast: false
     uses: ./.github/workflows/test_artifacts.yml
     with:
       amdgpu_families: ${{ inputs.amdgpu_families }}
       test_runs_on: ${{ inputs.test_runs_on }}
       artifact_run_id: ${{ inputs.artifact_run_id }}
+
+  build_portable_linux_python_packages:
+    needs: [build_portable_linux_artifacts]
```

I am aware that our CI is still flaky, but is it desired to change this to
```suggestion
    needs: [build_portable_linux_artifacts, test_linux_artifacts]
```
in the future to only proceed if tests passed? Might okay to build those in parallel to get a signal earlier though. If it should be a requirement please add a comment, otherwise ignore this comment.